### PR TITLE
Epsilon-scaled clipped versions of Softmax and Sigmoid as utilities

### PR DESCRIPTION
**Pull request by rohitsingh0812**
Blocking #107 

Epsilon-scaled clipped versions of Softmax and Sigmoid as utilities that can be used to avoid numerical unstability with Pytorch.nn Softmax and Sigmoid operations

## Tested

Added simple tests for both new classes.

### CODE REVIEW

**Review by fritzo — CHANGES_REQUESTED** [review]
Please fix lint errors:
```
./pyro/nn/clipped_nn.py:2:1: F401 'torch' imported but unused
./pyro/nn/clipped_nn.py:5:1: E302 expected 2 blank lines, found 1
./pyro/nn/__init__.py:5:1: F401 '.clipped_nn.ClippedSigmoid' imported but unused
./pyro/nn/__init__.py:5:1: F401 '.clipped_nn.ClippedSoftmax' imported but unused
./pyro/nn/__init__.py:5:55: W292 no newline at end of file
```

**Review by fritzo — APPROVED** [review]

**Review by jpchen — APPROVED** [review]
merging due to peer pressure
